examples/fhir/lab_order.json

```diff
@@ -0,0 +1,122 @@
+{
+  "resourceType": "Bundle",
+  "id": "df19c110-f28e-4277-9bec-95a9a7ec091e",
+  "type": "message",
+  "timestamp": "2023-02-24T09:55:17.713-06:00",
+  "entry": [
+    {
+      "resource": {
+        "resourceType": "MessageHeader",
+        "id": "a53d27ee-a78c-4395-99b5-5fd66e8b96ba",
+        "eventCoding": {
+          "system": "http://terminology.hl7.org/CodeSystem/v2-0003",
+          "code": "O21",
+          "display": "OML - Laboratory order"
+        },
+        "source": {
+          "name": "CDC Trusted Intermediary",
+          "endpoint": "https://reportstream.cdc.gov/"
+        }
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Patient",
+        "id": "infant-twin-1",
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race",
+            "extension": [
+              {
+                "url": "text",
+                "valueString": "Asian"
+              }
+            ]
+          }
+        ],
+        "identifier": [
+          {
+            "type": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                  "code": "MR",
+                  "display": "Medical Record Number"
+                }
+              ]
+            },
+            "value": "MRN7465737865"
+          }
+        ],
+        "name": [
+          {
+            "use": "official",
+            "family": "Solo",
+            "given": [
+              "Jaina"
+            ]
+          }
+        ],
+        "gender": "female",
+        "birthDate": "2017-05-15",
+        "_birthDate": {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/patient-birthTime",
+              "valueDateTime": "2017-05-15T11:11:00-05:00"
+            }
+          ]
+        },
+        "multipleBirthInteger": 1,
+        "contact": [
+          {
+            "relationship": [
+              {
+                "coding": [
+                  {
+                    "system": "http://terminology.hl7.org/CodeSystem/v2-0131",
+                    "code": "N",
+                    "display": "Next of kin"
```

Comment:
Would this be optional to include the next of kin of choice, or should the mother be included in the contact info? I'm wondering as I was looking at the example that FHIR.org has:

http://build.fhir.org/patient-example-infant-twin-1.json.html

Reply:
This is the correct way to add a next of kin, so we should put it here.